vital/types/metadata_traits.cxx

```diff
@@ -124,7 +171,34 @@ ::tag_to_name( vital_metadata_tag tag ) const
   return trait.name();
 }
 
-// ------------------------------------------------------------------
+// ----------------------------------------------------------------------------
+vital_metadata_tag
+metadata_traits
+::name_to_tag( std::string name ) const
+{
+  auto const& trait = find_name( name );
+  return trait.tag();
```

Comment:
My sort-of knee jerk reaction is to write:
```suggestion
  return find_name( name ).tag();
```

...but really, either way is fine. (I guess if you think the shorter version is better, the existing functions should also be changed, since it appears you were copying those.)